raiden/storage/sqlite.py

```diff
@@ -1474,6 +1475,19 @@ def update_light_client_protocol_message_set_signed_data(self, payment_id, msg_o
             last_id = cursor.lastrowid
         return last_id
 
+    def update_stored_msg_set_signed_tx_by_message_id(self, signed_tx: SignedTransaction, message_id: MessageID):
```

Comment:
I think this will need to be removed because of the changes made on the PR https://github.com/rsksmart/lumino/pull/128